src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -11,10 +11,9 @@
 import seedu.address.model.tag.Tag;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its tag name in the Wedlinker.
```

Comment:
Maybe can say 'Deletes a person identified using their displayed index in the Wedlinker and the tag name'